tst/standard/bipart.tst

```diff
@@ -484,8 +484,8 @@ gap> RightProjection(StarOp(x));
 
 # bipartition: Bipartition 1/3
 gap> Bipartition("test");
-Error, Semigroups: Bipartition: usage,
-the argument <classes> must consist of duplicate-free homogeneous lists,
```

Comment:
It looks like the changes in this pull request turn an informative error (the previous one) into an uninformative error (the current one). Please revert this.